src/com/facebook/buck/swift/SwiftCompileStep.java

```diff
@@ -52,11 +64,24 @@ public String getShortName() {
     return "swift compile";
   }
 
-  private ProcessExecutorParams makeProcessExecutorParams() {
+  private ProcessExecutorParams makeProcessExecutorParams() throws IOException {
```

Comment:
Do we need to handle this new possible exception anywhere?

Reply:
Not really, the exception will be re-threw in `execute` method and handled be its caller.